### ISSUE TITLE
Special characters don't work correctly

### ISSUE THREAD

**Issue by pedrofracassi** [SwitchbladeBot/invidget#14]
After Japanese support was added in #9, I've noticed that the renderer doesn't handle special characters correctly.

**Example:** `https://invidget.switchblade.xyz/discord-developers?language=ja`

![image](https://user-images.githubusercontent.com/25179120/90912197-8d254280-e3b0-11ea-8fbf-9953183cdff2.png)

**Comment by pedrofracassi** [comment]
Discord seems to use [Hiragino Sans](https://www.maisfontes.com/hiragino-sans-gb-w3) for japanese. We could try to use that as a fallaback for characters that Whitney doens't support.

**Comment by jaxxibae** [comment]
But does it affect all dialects (Hiragana, Katakana and Kanji)? @Flicksie

**Comment by pedrofracassi** [comment]
Discord seems to use a mix of hiragana and kanji (which should have been displayed in the screenshot above), so I guess it should affect Katakana too.

**Comment by jaxxibae** [comment]
Ah, ight. PS: that workaround should also work for server names

**Comment by pedrofracassi** [comment]
Yeah, I'll probably end up implementing some sort of global font fallback for all the text we render

**Comment by mome0320** [comment]
![image](https://user-images.githubusercontent.com/18746638/124532544-bbd2cd00-de4b-11eb-9291-4c40dc012c8e.png)
in Discord css. it classified by country.

but How about use Noto Sans for these?
i know it haven't in Discord Client but I think It covered.

**Comment by jaxxibae** [comment]
Noto Sans doesn't (at least) cover the Korean language, I think.
![image](https://user-images.githubusercontent.com/32089154/124566353-dfd9e280-de3a-11eb-8f3b-34fdbeac3c05.png)
It could cover other ones, though, but I didn't test it. Most probably we would have to implement Apple SD Gothic Neo, Hiragino Sans, YaHei New and JhengHei.

**Comment by Hoto-Cocoa** [comment]
> Noto Sans doesn't (at least) cover the Korean language, I think.
> ![image](https://user-images.githubusercontent.com/32089154/124566353-dfd9e280-de3a-11eb-8f3b-34fdbeac3c05.png)
> It could cover other ones, though, but I didn't test it. Most probably we would have to implement Apple SD Gothic Neo, Hiragino Sans, YaHei New and JhengHei.

Noto Sans has seperated CJK fontset now.
